configs/next/packages/pveclib/stage_1

```diff
@@ -0,0 +1,183 @@
+	# --includedir, respectively.
+	#
+	# On native builds, files are installed directly under the main prefix,
+	# whereas on cross builds, a '/usr' suffix is added to it.
+	if [[ "${cross_build}" == 'yes' ]]; then
+		configure_prefix="${dest_cross}/usr"
+	else
+		configure_prefix="${at_dest}"
+	fi
+	# The bin and lib directories depend on the word size, e.g.
+	# configure_bindir may be set to bin, bin32, or bin64.
+	configure_bindir=${configure_prefix}/$(find_build_bindir ${AT_BIT_SIZE})
+	configure_libdir=${configure_prefix}/$(find_build_libdir ${AT_BIT_SIZE})
+	# The variable configure_includedir is not actually required, because
+	# the default value provided by configure, i.e. ${prefix}/include, is
+	# exactly what we want. However, we prepare this variable for
+	# completeness purposes.
+	configure_includedir="${configure_prefix}/include"
+	# On the other hand, the variables configure_docdir, configure_infodir,
+	# and configure_mandir are indeed required, because the documentation
+	# files of all packages should be installed at the same place, i.e.:
+	#
+	#   ${at_dest}/share/doc/pkgname
+	#   ${at_dest}/share/info
+	#   ${at_dest}/share/man
+	#
+	# Notice that we cannot use the variable configure_prefix here, because
+	# it may contain the suffixes ppc/usr/ or ppc64le/usr/.
+	configure_docdir="${at_dest}/share/doc/pveclib"
+	configure_infodir="${at_dest}/share/info"
+	configure_mandir="${at_dest}/share/man"
+}
+
+atcfg_configure () {
+	# Select the prefixed compilation tools: gcc, g++, as, ld, ar, and
+	# ranlib, so that they work on both native and cross build.
+	configure_cc="${at_dest}/bin/${target64:-${target}}-gcc"
+	configure_cxx="${at_dest}/bin/${target64:-${target}}-g++"
+
+	# However, it also requires to run configure from a different
+	# directory.  So we're forced to duplicate source code and build in
+	# another directory (build/ in this case).
+	rm -rf build/
+	mkdir build/
+	pushd build > /dev/null
+
+	if [[ "${cross_build}" == 'yes' ]]; then
+		export ac_cv_func_malloc_0_nonnull=yes
+	fi
+
+	PATH=${at_dest}/bin:${PATH} \
+	CC="${configure_cc} -m${AT_BIT_SIZE}" \
+	CXX="${configure_cxx} -m${AT_BIT_SIZE}" \
+	CFLAGS="-g -O" \
+	CXXFLAGS="-g -O" \
+		../configure --build=${configure_build} \
+			     --host=${configure_host} \
+			     --target=${configure_target} \
+			     --prefix=${configure_prefix} \
+			     --bindir=${configure_bindir} \
+			     --libdir=${configure_libdir} \
+			     --docdir=${configure_docdir} \
+			     --infodir=${configure_infodir} \
+			     --mandir=${configure_mandir} \
+			     --includedir=${configure_includedir} \
+		|| return ${?}
+	popd > /dev/null
+}
+
+atcfg_make () {
+	pushd build > /dev/null
+	# We're forced to use -f in order to work around an issue with the
+	# mtime value of the files.  As git doesn't preserve it, GNU make
+	# thinks that it must re-generate the Makefile upon every execution
+	# because some m4 files do have the same mtime of Makefile.im.
```

Comment:
This should not be necessary.
Is this a bug upstream?  If it is, we need to report it.